packages/packages.go

```diff
@@ -30,24 +22,7 @@ type Packages interface {
 // convertGopkgNameToPurl will convert the Gopkg name into a Package URL
 //
 // FIXME: Research the various Gopkg name formats and convert them correctly
-func convertGopkgNameToPurl(name string) string {
-	switch {
-	case githubPattern.MatchString(name):
-		// Currently OSS Index's github format support is based on repository
-		// owner/name, so restrict the PURL name path.
-		//
-		// Once golang format support is improved we can switch to that and get
-		// more precise and refined results on repository sub-paths
-		name = githubPattern.ReplaceAllString(name, "github/$1/$2")
-
-	case gopkg2Pattern.MatchString(name):
-		name = gopkg2Pattern.ReplaceAllString(name, "github/$1/$2")
-
-	case gopkg1Pattern.MatchString(name):
-		name = gopkg1Pattern.ReplaceAllString(name, "github/go-$1/$1")
-
-	default:
-		name = "golang/" + name
-	}
-	return name
+func convertGopkgNameToPurl(name string) (rename string) {
+	rename = "golang/" + name
+	return
```

Comment:
I would still like to use the regular expressions to force `gopkg.in` URLs to expand to the appropriate `github.com` namespace.